### DISCUSSION TITLE
KAMP/Adaptive Purge line intersects Tree(auto) base layers skirt brim

### DISCUSSION THREAD

**Discussion by living-relation** [Guilouz/Creality-Helper-Script-Wiki#704] [Issues]
when using adaptive purge line, my hotend nozzle ran into my adaptive purge line towards the front of the bed. Using auto Tree support with Slim style, when printing the bottom round interface loops.
![20241120_213030](https://github.com/user-attachments/assets/0499c6ae-f0c2-4d76-97d8-65b3e42042c6)

**Comment by waynevanrooyen1971** [comment]
Yes i am having same issue dont know where to report this either.

**Comment by KiLLie2k** [comment]
I can help

in the folder Helper-Script/KAMP there is a file kamp_settings.cfg there is a settings that you can adjust for where the purge line should be.

is you use supports that have a 15mm brim change the settings to 20 etc.

this is the setting

variable_purge_margin: 10                   # Distance the purge will be in front of the print area. Default is 10.

**Comment by waynevanrooyen1971** [comment]
Thank you


***@***.***
WAYNE VAN ROOYEN
Shopify Expert And Partner
  [Title: Facebook icon] <https://www.facebook.com/ShoptifiedSA>
Mobile: 0825550096  Email: ***@***.******@***.***/>

Web: http://www.shoptified.com<http://www.shoptified.com/> Address: 400 Clark Road Glenwood, Durban, 4001
